app/views/application/_thumbnail_core.html.haml

```diff
@@ -1,9 +1,14 @@
 %a.card{:href => "#{eval("#{@thumb_obj[:link]}_path(#{@thumb_obj[:id]})")}"}
   .caption
     %h5= @thumb_obj[:name]
-    - if @thumb_obj[:image].present?
-      .image-wrapper
-        = image_tag @thumb_obj[:image]
+    - if Rails.env.production?
```

Comment:
This pattern gets repeated a lot, so it may be a natural choice to refactor into a method/methods on the model. Also would keep with the spirit of keeping presentation work in views and logic in models.

I would make a .has_image? method that checks the right field depending on env, and then a .image_url method that does similar logic.